app/jobs/indexer.rb

```diff
@@ -1,6 +1,7 @@
 class Indexer < ApplicationJob
   extend StatsD::Instrument
   include TraceTagger
+  include SemanticLogger::Loggable
```

Comment:
Ditto for the ApplicationJob logger so we don't have to include the semantic module in each individual job class, I think?